integration/testdata/vendored/vendor/github.com/BurntSushi/toml/decode.go

```diff
@@ -3,8 +3,8 @@ package toml
 import (
 	"fmt"
 	"io"
-	"io/ioutil"
 	"math"
+	"os"
```

Comment:
Were the changes in this vendored dependency intentional?

Reply:
Oh, no, no they we not. Let me change that back.